### PR TITLE
Endpoint refactor

### PR DESCRIPTION
**Pull request by CreeperBeatz**
- login token validity now 7 days
- Refactor stamps functionality
- rename get_stamp_token to get_id_token
- token now only takes user id
- receive_stamp now called make_stamp
- make stamp only accessible by employees

### CODE REVIEW

File: src/stamps.py [hunk 1]
```diff
@@ -30,27 +32,43 @@ class InvalidStampToken(Exception):
...
+
     try:
         # Decrypt the data from the token
         token: List[str] = __dec_box.decrypt(bytes.fromhex(token)) \
             .decode('utf-8').split('\n')
 
         logger.debug("Token decoded : {token}")
         logger.debug(f"time now :  {datetime.utcnow()}")
-        logger.debug(f"token time: {datetime.fromisoformat(token[2])}")
+        logger.debug(f"token time: {datetime.fromisoformat(token[1])}")
```
Comment (KirilMihaylov) [inline src/stamps.py]: Safe with the current implementation but far from ideal to be before the length check.

File: src/response_formaters.py [hunk 2]
```diff
@@ -45,17 +45,24 @@ async def get_tourist_sites(session: AsyncSession, site_type: AllSitesFilter, vi
...
+        # Fetch image id
+        image_id = (
+            await Images.all_by_place(
+                session, site.get('id'),
+            )
+        )
+
+        # Get first element of array only after we know we have an image
+        if image_id is not None:
+            image_id = image_id[0]
```
Comment (KirilMihaylov) [inline src/response_formaters.py:57]: Not length checked before usage! The method will **always** return a list but it **can** be empty!

**Review by KirilMihaylov — CHANGES_REQUESTED** [review]

File: src/response_formaters.py (hunk 2 above)
Comment (CreeperBeatz) [reply src/response_formaters.py:57]: A proper fix will be a request that gets the first element only

Comment (CreeperBeatz) [reply src/response_formaters.py:57]: I will include it in the TODO list

**Review by KirilMihaylov — APPROVED** [review]

**Pull request merged**